Player/WalkthroughRunner.vb

```diff
@@ -39,11 +43,15 @@
                     End If
                 ElseIf cmd.StartsWith("label:") Then
                     ' ignore
+                ElseIf cmd.StartsWith("delay:") Then
+                    delay = Int32.Parse(cmd.Substring(6).Trim)
                 ElseIf cmd.StartsWith("event:") Then
                     Dim eventData As String() = cmd.Substring(6).Split(New Char() {";"c}, 2)
                     Dim eventName As String = eventData(0)
                     Dim param As String = eventData(1)
                     m_game.SendEvent(eventName, param)
+                ElseIf cmd.StartsWith("runtime:") Then
+                    showRuntime = True

```

Comment:
If the expectation is that people use `runtime:` at the end of the walkthrough, then maybe this is where the code for displaying the runtime should be too? No need for a separate `showRuntime` boolean then, and it would mean you could use the `runtime:` command at multiple points in the walkthrough?